packages/react/src/migrations/update-18-0-0/add-mf-env-var-to-target-defaults.ts

```diff
@@ -8,10 +8,11 @@ import {
 import { addMfEnvToTargetDefaultInputs } from '../../utils/add-mf-env-to-inputs';
 
 export default async function (tree: Tree) {
-  if (!hasModuleFederationProject(tree)) {
+  const bundler = hasModuleFederationProject(tree);
```

Comment:
Should there be another migration for 20.4 for projects created after v18?

Reply:
There is :) 
https://github.com/nrwl/nx/pull/29584/files#diff-7752a3351af4cb540782fcaf8e3f320256efd9abc08ff467b466f9ec6b3b7402R50-R55